src/python/CRABClient/Commands/remote_copy.py

```diff
@@ -140,10 +140,13 @@ def __call__(self):
                 continue
 
             ##### Creating the command
+            # better to execut grid commands in the pre-CMS environment
+            undoScram = "which scram >/dev/null 2>&1 && eval `scram unsetenv -sh`"
+            cmd = undoScram + cmd
```

Comment:
`cmd` is not defined before, any reason to add `+ cmd`?
Anyhow it is overridden by https://github.com/dmwm/CRABClient/pull/4758/files#diff-62f68c1aa854c51b75d7304faaff2af7R153 and not used before.

Reply:
thanks @lecriste  for spotting this. It was a plain mistake.  But I do not understand what you write about it being overridden. I do not understand if I should fix this or you did somethig already, or ... ?

Reply:
`cmd` is assigned in L145 and reassigned in L153 without being used before, so the former value is never used.
I did not touch the code.

Reply:
thanks, understood. never the less L145 will cause a failure ! I will fix this.

Reply:
also the way is is coded now will plainly not work !